editor/src/components/editor/editor-component.tsx

```diff
@@ -505,7 +505,7 @@ const OpenFileEditor = betterReactMemo('OpenFileEditor', () => {
     const openEditorTab = getOpenEditorTab(store.editor)
     return {
       noFileOpen: openEditorTab == null,
-      isUiJsFileOpen: selectedFile != null && isUIJSFile(selectedFile),
+      isUiJsFileOpen: selectedFile != null && isTextFile(selectedFile),
```

Comment:
I wonder if this (and other places that were assuming a `UIJSFile` previously) should also check that the parse status isn't `Unparsed` :thinking:

Reply:
Done in adc65a8